rich/markdown.py

```diff
@@ -405,16 +413,24 @@ class Markdown(JupyterMixin):
     """
 
     elements: ClassVar[Dict[str, Type[MarkdownElement]]] = {
-        "paragraph": Paragraph,
-        "heading": Heading,
+        "paragraph_open": Paragraph,
+        "heading_open": Heading,
+        "fence": CodeBlock,
         "code_block": CodeBlock,
-        "block_quote": BlockQuote,
-        "thematic_break": HorizontalRule,
-        "list": ListElement,
-        "item": ListItem,
+        "blockquote_open": BlockQuote,
+        "hr": HorizontalRule,
+        "bullet_list_open": ListElement,
+        "ordered_list_open": ListElement,
+        "list_item_open": ListItem,
         "image": ImageItem,
     }
-    inlines = {"emph", "strong", "code", "strike"}
+
+    # Maps tag names to Rich style keys, if tag and key differ
+    tag_to_style_name = {
+        "em": "emph",
+    }
+
```

Comment:
I could probably just add a `markdown.em` style and have it match `markdown.emph`, then get rid of this. When I made it I thought there would be more mismatches 🤷

Reply:
I ended up making this change